QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/services/QueryService.java

```diff
@@ -181,6 +181,7 @@ private Uni<QueryResult> handleEntityMap(Long resultCount, List<EntityMapEntry>
 
 		if (unis.isEmpty()) {
 			QueryResult q = new QueryResult();
+			if(count) q.setCount(resultCount);
```

Comment:
add curly brackets here. if statements with brackets are generally considered bad practice because of maintainability.

Reply:
done changes as suggested

Reply:
Test cases will be added soon in PreviousValue-MergePatch branch